cmd/zetae2e/config/clients.go

```diff
@@ -0,0 +1,120 @@
+package config
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+
+	"github.com/btcsuite/btcd/rpcclient"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/zeta-chain/zetacore/contrib/localnet/orchestrator/smoketest/config"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	fungibletypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// getClientsFromConfig get clients from config
+func getClientsFromConfig(ctx context.Context, conf config.Config, evmPrivKey string) (
+	*rpcclient.Client,
+	*ethclient.Client,
+	*bind.TransactOpts,
+	crosschaintypes.QueryClient,
+	fungibletypes.QueryClient,
+	authtypes.QueryClient,
+	banktypes.QueryClient,
+	observertypes.QueryClient,
+	*ethclient.Client,
+	*bind.TransactOpts,
+	error,
+) {
+	btcRPCClient, err := getBtcClient(conf.RPCs.Bitcoin)
+	if err != nil {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, err
+	}
+	goerliClient, goerliAuth, err := getEVMClient(ctx, conf.RPCs.EVM, evmPrivKey)
+	if err != nil {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, err
+	}
+	cctxClient, fungibleClient, authClient, bankClient, observerClient, err := getZetaClients(conf.RPCs.ZetaCoreGRPC)
+	if err != nil {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, err
+	}
+	zevmClient, zevmAuth, err := getEVMClient(ctx, conf.RPCs.Zevm, evmPrivKey)
+	if err != nil {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, err
+	}
+	return btcRPCClient,
+		goerliClient,
+		goerliAuth,
+		cctxClient,
+		fungibleClient,
+		authClient,
+		bankClient,
+		observerClient,
+		zevmClient,
+		zevmAuth,
+		nil
+}
+
+// getBtcClient get btc client
+func getBtcClient(rpc string) (*rpcclient.Client, error) {
+	connCfg := &rpcclient.ConnConfig{
+		Host:         rpc,
+		User:         "smoketest",
+		Pass:         "123",
+		HTTPPostMode: true,
+		DisableTLS:   true,
+		Params:       "testnet3",
+	}
+	return rpcclient.New(connCfg, nil)
+}
```

Comment:
When running the e2e tester on a live network is there a way to pass in a different UN/PW?

Reply:
Not sure what it has been resolved but we can put these values in the config file. I started work on the e2e tester specifically for the EVM now but not yet ready.

Reply:
I see the other issues for them now. https://github.com/zeta-chain/node/issues/1482
Thanks!